nilearn/plotting/surface/surf_plotting.py

```diff
@@ -1011,94 +1006,27 @@ def plot_surf_roi(
 
     nilearn.surface.vol_to_surf : For info on the generation of surfaces.
     """
-    # set default view to dorsal if hemi is both and view is not set
-    check_params(locals())
-    if view is None:
-        view = "dorsal" if hemi == "both" else "lateral"
-
-    roi_map, surf_mesh, bg_map = check_surface_plotting_inputs(
-        roi_map, surf_mesh, hemi, bg_map
-    )
-
-    if engine == "matplotlib" and avg_method is None:
-        avg_method = "median"
```

Comment:
Is there a special reason that `avg_method=median` for `plot_surf_roi` or it was a random choice? 

In current implementation, I leave `plot_surf` to adjust this value, which sets `avg_method=mean`. If it is convenient, I'll leave the code as is, otherwise I'll update to set it as `median`.

Reply:
I don't remember and I think the choice was made before I joined the team, so not sure.
I'd suggest leaving it as is.

Reply:
I just want to confirm that leaving as is, I meant `avg_method=mean` set by `plot_surf` which is different than `avg_method=median` in original code for `plot_surf_roi`.